transpose_comments.py

```diff
@@ -184,7 +221,13 @@ def extract_comment(textbox_content):
     return textbox_content
     
 
-def alternate_texts_and_images(doc, textbox_content):
+def alternate_texts_and_images(doc: DocxTemplate, textbox_content: str) -> list:
+    """
+    Comments may contain pictures. To keep them, we create a liste to store text and image path
+    
```

Comment:
... The image size is fixed since it cannot be extracted from the initial document.